index.html

```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+             <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about-section">
+        <div class="about-me">
+            <h1>About me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <button class="get-resume">Get my resume</button>
+        </div>
```

Comment:
- [ ] Kindly make sure the `about me` heading sits in one line. You are expected to stick to design as much as possible.
- [ ] Kindly reduce the size of the button to match the figma template.
- [ ] Kindly bring the background image a bit downwards to match the given template
- [ ] Please use the right font family for the text. You are expected to use the same font as the design.
### Your design
![image](https://user-images.githubusercontent.com/94551484/227733997-558609ec-64c3-4990-b79d-f3502a5ec123.png)
### FIgma
![image](https://user-images.githubusercontent.com/94551484/227734031-0e3cfc6d-4077-4488-b456-43a415d92173.png)